barbican/cmd/barbican_manage.py

```diff
@@ -24,6 +24,9 @@
 
 from oslo_config import cfg
 from oslo_log import log as logging
+from oslo_config import cfg as oslo_cfg
```

Comment:
Imports are not ordered alphabetically. This is also already imported on lin 25 under a different name. It would be better to not create 2 names for the same thing.